Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic

```diff
@@ -7,22 +7,22 @@
     <widget-library name="glade-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
```

Comment:
Sorry I did not realize that these were included. I am going to go fix this merge.